community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v2_3/GeneratedQueryStructure.scala

```diff
@@ -766,6 +773,7 @@ private object Methods {
   val nodesGetAll = method[ReadOperations, PrimitiveLongIterator]("nodesGetAll")
   val nodeGetProperty = method[ReadOperations, Object]("nodeGetProperty")
   val nodesGetFromIndexLookup = method[ReadOperations, PrimitiveLongIterator]("nodesGetFromIndexLookup", typeRef[IndexDescriptor], typeRef[Object])
+  val nodeGetUniqueFromIndexLookup = method[ReadOperations, Long]("nodeGetUniqueFromIndexLookup", typeRef[IndexDescriptor], typeRef[Object])
```

Comment:
I prefer `nodeGetFromUniqueIndexLookup` here, following the same thought process from above.